README.md

```diff
@@ -81,6 +81,9 @@ Here are the details of available options.
 
 | Parameter                 | Description                                                                              |
 |---------------------------|------------------------------------------------------------------------------------------|
+| DISABLE\_ADMINS           | If 'true', admins are disabled meaning DEBUG\_GRANTED\_TO is not taken into account      |
+|                           | anymore and only super admins can log into containers. This feature is meant to remove   |
+|                           | authorizations in  production environments.                                              |
```

Comment:
Could we implement it the other way around: custom privileges through *DEBUG_GRANTED_TO* label to be opt-in, so disabled by default?

Reply:
This is indeed a good idea, I propose to change DISABLE_ADMINS into ENABLE_PER_APP_ADMINS. Looks good?

Reply:
👍